<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>886</width>
    <height>701</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Marche</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../res/marche.qrc">
    <normaloff>:/marche/logo-new.svg</normaloff>:/marche/logo-new.svg</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <property name="margin">
     <number>0</number>
    </property>
    <property name="spacing">
     <number>0</number>
    </property>
    <item row="0" column="0">
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QWidget" name="layoutWidget">
       <layout class="QVBoxLayout" name="leftlayout">
        <item>
         <widget class="QWidget" name="cachePanel" native="true">
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <property name="margin">
            <number>0</number>
           </property>
           <item>
            <spacer name="horizontalSpacer_2">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>180</width>
               <height>19</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QLabel" name="lblCached">
             <property name="styleSheet">
              <string notr="true">color: rgb(255, 0, 0);</string>
             </property>
             <property name="text">
              <string>Cached credentials:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="lblCachedUserCreds">
             <property name="styleSheet">
              <string notr="true">color: rgb(255, 0, 0);</string>
             </property>
             <property name="text">
              <string>{username}</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QToolButton" name="clearCredBtn">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="toolTip">
              <string>Clear cached credentials</string>
             </property>
             <property name="styleSheet">
              <string notr="true"/>
             </property>
             <property name="text">
              <string>X</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QListWidget" name="hostList">
          <property name="minimumSize">
           <size>
            <width>200</width>
            <height>0</height>
           </size>
          </property>
          <property name="contextMenuPolicy">
           <enum>Qt::CustomContextMenu</enum>
          </property>
          <property name="iconSize">
           <size>
            <width>16</width>
            <height>24</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="surface" native="true">
       <property name="minimumSize">
        <size>
         <width>410</width>
         <height>610</height>
        </size>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="margin">
         <number>0</number>
        </property>
        <item>
         <widget class="QLabel" name="logoLabel">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="pixmap">
           <pixmap resource="../res/marche.qrc">:/marche/logo-watermark.png</pixmap>
          </property>
          <property name="scaledContents">
           <bool>false</bool>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>886</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="separator"/>
    <addaction name="actionAdd_host"/>
    <addaction name="actionAdd_network"/>
    <addaction name="actionScan_local_network"/>
    <addaction name="separator"/>
    <addaction name="actionSave_session_as"/>
    <addaction name="actionLoad_session"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionPreferences"/>
   </widget>
   <widget class="QMenu" name="menuJobs">
    <property name="title">
     <string>Jobs</string>
    </property>
    <addaction name="actionReload"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuJobs"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextBesideIcon</enum>
   </property>
   <property name="floatable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionAdd_host"/>
   <addaction name="actionAdd_network"/>
   <addaction name="actionScan_local_network"/>
   <addaction name="actionReload"/>
  </widget>
  <widget class="QStatusBar" name="mainStatusBar"/>
  <action name="actionAdd_host">
   <property name="icon">
    <iconset resource="../res/marche.qrc">
     <normaloff>:/marche/server.png</normaloff>:/marche/server.png</iconset>
   </property>
   <property name="text">
    <string>Add host</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+H</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="icon">
    <iconset resource="../res/marche.qrc">
     <normaloff>:/marche/door-open.png</normaloff>:/marche/door-open.png</iconset>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="icon">
    <iconset resource="../res/marche.qrc">
     <normaloff>:/marche/logo-new.svg</normaloff>:/marche/logo-new.svg</iconset>
   </property>
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionScan_local_network">
   <property name="icon">
    <iconset resource="../res/marche.qrc">
     <normaloff>:/marche/server-cast.png</normaloff>:/marche/server-cast.png</iconset>
   </property>
   <property name="text">
    <string>Scan local network</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+B</string>
   </property>
  </action>
  <action name="actionReload">
   <property name="icon">
    <iconset resource="../res/marche.qrc">
     <normaloff>:/marche/arrow-circle-225.png</normaloff>:/marche/arrow-circle-225.png</iconset>
   </property>
   <property name="text">
    <string>Reload jobs</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
  <action name="actionLoad_session">
   <property name="icon">
    <iconset resource="../res/marche.qrc">
     <normaloff>:/marche/folder-open.png</normaloff>:/marche/folder-open.png</iconset>
   </property>
   <property name="text">
    <string>Load session</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionSave_session_as">
   <property name="icon">
    <iconset resource="../res/marche.qrc">
     <normaloff>:/marche/disk.png</normaloff>:/marche/disk.png</iconset>
   </property>
   <property name="text">
    <string>Save session as</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionPreferences">
   <property name="icon">
    <iconset resource="../res/marche.qrc">
     <normaloff>:/marche/drill.png</normaloff>:/marche/drill.png</iconset>
   </property>
   <property name="text">
    <string>Preferences</string>
   </property>
  </action>
  <action name="actionAdd_network">
   <property name="icon">
    <iconset resource="../res/marche.qrc">
     <normaloff>:/marche/server-cast.png</normaloff>:/marche/server-cast.png</iconset>
   </property>
   <property name="text">
    <string>Add network</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../res/marche.qrc"/>
 </resources>
 <connections/>
</ui>
